### PR TITLE
ansible 4.1.0

### PR DESCRIPTION
**Pull request by mr-deamon**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/ansible.rb [hunk 1]
```diff
@@ -578,7 +578,6 @@ class Ansible < Formula
     url "https://files.pythonhosted.org/packages/e3/ed/2092731880f0de5b07067fc446dc0fc5166f2ee98018b6d524cd3e28a69d/zabbix-api-0.5.4.tar.gz"
     sha256 "2d6c62001cb79a7de6fe286424967276edaca09d3833b72fb04f7863f29fce4b"
   end
-
```
Comment (nandahkrishna) [inline Formula/ansible.rb:581]: Could you add this (blank) line back?

Comment (mr-deamon) [reply Formula/ansible.rb]: done. sorry

Comment (nandahkrishna) [reply Formula/ansible.rb]: No problem!

**Review by nandahkrishna — APPROVED** [review]
Thanks for your first contribution to Homebrew, @mr-deamon! 🍻

**Pull request closed without merging**